tensorflow_text/python/ops/wordpiece_tokenizer.py

```diff
@@ -58,13 +59,17 @@ def __init__(self,
         `tf.int64`, the `vocab_lookup_table` is used to convert the
         `unknown_token` to an integer. If this is set to `None`,
         out-of-vocabulary tokens are left as is.
+      split_unknown_characters: (optional) Whether to split out single unknown
```

Comment:
But it's not just suffices that would get split, right?
E.g. it might map "gooo🌕🌕oogle" to ["gooo", "🌕", "🌕", "oogle"] if "🌕" isn't a known subword token. (whereas before the whole thing would be UNK). I don't think "🌕" is an "unknown suffix" here but rather an "unknown character" right?

Reply:
also it could be at the start of the word "🌕MG" -> "🌕", "##MG" - so it is not necessarily a suffix in this sense either

Reply:
ah, I see. my bad I misunderstood and thought that you would stop early and turn the entire suffix into an [UNK] token.